### DISCUSSION TITLE
How do i create a multi-tenancy database.

### DISCUSSION THREAD

**Discussion by jcesarg09** [AntlerVC/firetable#1150] [Support / Q&A] [answered]
I need to create a multi-tenancy app, where companies signup and each company has their admin and users and data. Can this be achieve in rowy? Im using flutterflow and firebase. I asked google and chatgpt and both say its possible by using subcollections on each master collection under company. Do anyone know how?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @jcesarg09. Yes, you can achieve this by creating sub-tables (subcollections). I am assuming you have a collection named, let's say **Companies**. Inside this collection, you have multiple documents, each for a company that signs up. 

In Rowy, we have a row for every document. Now, to create a subcollection, you can add a new column, and set its field type to **Sub Table**. Here's the link to the documentation on how you can utilize Sub Tables: https://docs.rowy.io/field-types/sub-table. Hope this helps!
